<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SheZhi</class>
 <widget class="QMainWindow" name="SheZhi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>卧室</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="WoShi"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>阳台</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="YangTai"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>厨房</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ChuFang"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>卫生间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="WC"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ZJM_BT">
        <property name="text">
         <string>主界面</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
